### PR TITLE
Add usage instructions

### LINKED ISSUE #32: Improve instructions for plugin usage
We need to add some more instructions to the README about how a user can use our plugin. 

**Prerequisites:**
1. rad auth + rad clone
2. git clone + rad auth + rad init
3. git init + rad auth + rad init 

**Contribute scenario:**  
* open radicle-initialised repo in IntelliJ
* make some changes, git add, git commit -m 
* rad push (from menu, toolbar) OR git push rad + rad sync 

**Maintainer scenario:** 
* open radicle-initialised repo in IntelliJ
* rad pull 
* merge branch (??)

### PR DESCRIPTION
**Pull request by gsaslis**
Fixes #32

### CODE REVIEW

**Comment by gsaslis** [comment]
@JChrist I think @Stelios123 had something to add here. Let's wait for that before merging this in.

**Comment by Stelios123** [comment]
**Windows users with wsl**

- Open your linux distro command line
- Install keychain `sudo apt install keychain`
- Then add the following line to your shell configuration (~/.bashrc or ~/.zshrc) 
   eval &#96;keychain --quiet --eval --agents ssh id_rsa&#96;
- Run `rad auth` in your project directory

You have to run `rad auth` every time you booting wsl

**Comment by gsaslis** [comment]
@JChrist / @Stelios123 ok, I added a commit, so this is now ready for review

File: README.md [hunk 1]
```diff
@@ -26,6 +26,39 @@ It requires the Git plugin, as well as [radicle cli](https://radicle.network/get
...
+
+* A Radicle-initialised Git repo
+* The `rad` [Command Line Interface (CLI) tool](https://github.com/radicle-dev/radicle-cli) installed
+* A configured Radicle identity, with `rad auth`. Follow the [Radicle CLI quick start guide](https://radicle.network/get-started.html) for more info. 
+
+
+* On Windows: 
+  * Windows Subsystem for Linux (WSL) 2
+  * Keychain 2.8.5 (e.g. with `sudo apt-get install keychain` or `sudo dnf install keychain` )
+  * Add the following line to your shell configuration (~/.bashrc or ~/.zshrc)
```
Comment (JChrist) [inline README.md:41]:
```suggestion
  * Add the following line to your bash shell configuration `~/.bashrc`
```

**Review by Stelios123 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -26,6 +26,39 @@ It requires the Git plugin, as well as [radicle cli](https://radicle.network/get
...
+
+### Prerequisites
+
+* A Radicle-initialised Git repo
+* The `rad` [Command Line Interface (CLI) tool](https://github.com/radicle-dev/radicle-cli) installed
+* A configured Radicle identity, with `rad auth`. Follow the [Radicle CLI quick start guide](https://radicle.network/get-started.html) for more info. 
+
+
+* On Windows: 
+  * Windows Subsystem for Linux (WSL) 2
```
Comment (JChrist) [inline README.md:39]:
```suggestion
  * Windows Subsystem for Linux (WSL) 2 (The following steps are required to be performed in your default WSL2 distribution)
```

**Review by JChrist — CHANGES_REQUESTED** [review]

**Pull request merged**